cfgov/ask_cfpb/models/pages.py

```diff
@@ -294,14 +294,6 @@ def get_heading(self):
         else:
             return self.portal_topic.title(self.language)
 
-    def get_context(self, request, *args, **kwargs):
```

Comment:
FYI @higs4281, I've removed this code from Ask pages as duplicative because the default CFGOVPage now `activate`s on every Wagtail page.

Reply:
We previously couldn't rely on the CFGOVPage's activation, because it happens in its [serve method](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L239-L250), which routable pages do not use.

Does the middleware change this? If so, can we remove the activation code from CFGOVPage's serve method?

Reply:
When I test this locally, we lose Spanish labeling on Spanish portal search pages, which use RoutablePagemixin.

<img width="1251" alt="loss_of_spanish" src="https://user-images.githubusercontent.com/515885/84685231-ecc22600-af07-11ea-991b-c5c6d625e6ec.png">

Reply:
That doesn't happen on master.

Reply:
<img width="1228" alt="no_es_loss" src="https://user-images.githubusercontent.com/515885/84685494-66f2aa80-af08-11ea-9b83-4dfdbc10264c.png">

Reply:
Ah, shoot. Nice catch. Let me take another look at this.

Reply:
Thanks @higs4281. I ended up backing out that commit from this PR, so Ask pages should work right again. It's unfortunate that there doesn't seem to be a generic way to introduce common serve logic to RoutablePages. https://github.com/wagtail/wagtail/pull/6128 further codifies that RoutablePages should act like Django views, and shouldn't try to hook back into `Page.serve`.